src/site/static/eligibility.liquid

```diff
@@ -1954,9 +1954,42 @@ pageClass: "page-public-assistance"
       meetsDischargeReq);
   }
 
+  // GA-specific references:
+  //   https://socialservices.sccgov.org/about-us/department-employment-and-benefit-services/regulation-and-policy-handbooks/general-assistance
+  //   https://stgenssa.sccgov.org/debs/policy_handbook_Charts/ch-ga.pdf
+  //
+  // TO DO:
+  //   https://stgenssa.sccgov.org/debs/policy_handbook_GA/gachap07.pdf (7. Citizens/Noncitizens)
+  //     Qualified non-citizen: Need to determine all non-citizen groups which fit this category. May or may not need to add more detail to immigration status form page.
+  //
+  //   https://stgenssa.sccgov.org/debs/policy_handbook_Charts/ch-ga.pdf (Section 4.2)
+  //     Implement Grant level - UNSHARED Housing in Section 4.2.1. Also, max gross income cannot exceed max grant level, and max grant level changees with family size and living arrangement.
+  //
+  //   https://stgenssa.sccgov.org/debs/policy_handbook_GA/gachap05.pdf (Section 5.1)
+  //     Maximum age can be over 64 years with some conditions. May or may not need to implement this.
   function gaEligible() {
-    // TODO
-    return false;
+    const age = document.getElementById("age").value || null;
```

Comment:
No need to change it for this PR, but FYI:  There is a function called `getValueOrNull()` that you can use here.

```
const age = getValueOrNull('age');
```